README.md

```diff
@@ -66,32 +80,47 @@ list.template = {
 };
 ```
 
-Updates to the `items` array instance will not be captured by `<virtual-list>`.
+## Data manipulation
 
-Either set a new array to trigger the update, or use `requestReset()` to notify of changes if you want to keep the same array instance.
+`<virtual-list>` will rerender the displayed items when it receives a new `items` array.
 ```js
 list.items = list.items.concat([{name: 'new item'}]);
+```
 
+If you want to keep the same `items` array instance, use `requestReset()` to rerender the displayed items.
+```js
 list.items.push({name: 'new item'});
+list.items[0].name = 'item 0 changed!';

```

Comment:
Does this version work? Does it just rerender everything currently being displayed?

Reply:
Yes, it will rerender all the displayed items. If the user does recycling this translates to a bunch of array pop()/push() + updateChild()